mailbox/event/json/src/main/scala/org/apache/james/event/json/MetaDataDTO.scala

```diff
@@ -0,0 +1,89 @@
+/** **************************************************************
+  * Licensed to the Apache Software Foundation (ASF) under one   *
+  * or more contributor license agreements.  See the NOTICE file *
+  * distributed with this work for additional information        *
+  * regarding copyright ownership.  The ASF licenses this file   *
+  * to you under the Apache License, Version 2.0 (the            *
+  * "License"); you may not use this file except in compliance   *
+  * with the License.  You may obtain a copy of the License at   *
+  * *
+  * http://www.apache.org/licenses/LICENSE-2.0                 *
+  * *
+  * Unless required by applicable law or agreed to in writing,   *
+  * software distributed under the License is distributed on an  *
+  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+  * KIND, either express or implied.  See the License for the    *
+  * specific language governing permissions and limitations      *
+  * under the License.                                           *
+  * ***************************************************************/
+
+package org.apache.james.event.json
+
+import java.time.Instant
+import java.util.Date
+
+import javax.mail.{Flags => JavaMailFlags}
+import org.apache.james.mailbox.model.{MessageId, MessageMetaData => JavaMessageMetaData}
+import org.apache.james.mailbox.{FlagsBuilder, MessageUid}
+
+object MetaDataDTO {
```

Comment:
As discussed with @mbaechler , we came to the conclusion having a separated file for DTOs that are not Event was a good idea (but not one file per DTO). Can you group them in one `DTOs.scala` file?